### ISSUE TITLE
Accessibility violation on the problems panel

### ISSUE THREAD

**Issue by jarekdanielak** [camunda/camunda-modeler#4446]
### Describe the bug

If you close the bottom panel on Problems tab, with "No problems found", the [Accessibility Checker plugin](https://github.com/bpmn-io/camunda-modeler-plugin-a11y-checker) will report a violation: `Scrollable region must have keyboard access'`.

https://github.com/user-attachments/assets/68f8ca89-c81f-4d2b-b6bf-6252ede3bf9c

### Steps to reproduce

1. Create a new BPMN/DMN Diagram or Form.
2. Make sure the bottom panel is open on Problems tab and there are no problems found.
3. Close the panel.
4. Check Accessibility Violations.

### Expected behavior

No violation when closing the panel.

### Environment

- Camunda Modeler Version: 5.26.0-rc.0
- Installed plug-ins: [camunda-modeler-plugin-a11y-checker](https://github.com/bpmn-io/camunda-modeler-plugin-a11y-checker)
